install/install.php

```diff
@@ -112,18 +87,15 @@ public static function get_default_options() {
 	 * @return void
 	 */
 	protected function _activate() {
-		if ( $options = get_option( 'polylang' ) ) {
-			// Check if we will be able to upgrade
-			if ( version_compare( $options['version'], POLYLANG_VERSION, '<' ) ) {
-				$upgrade = new PLL_Upgrade( $options );
-				$upgrade->can_activate();
-			}
-		}
-		// Defines default values for options in case this is the first installation
-		else {
-			update_option( 'polylang', self::get_default_options() );
+		$options = $this->get_options();
+
+		if ( version_compare( $options->get( 'version' ), POLYLANG_VERSION, '<' ) ) {
+			// Check if we will be able to upgrade.
+			( new PLL_Upgrade( $options ) )->can_activate();
 		}
 
+		$this->save_options( $options );
```

Comment:
Are we sure the default ones are saved here? 🤔

Reply:
Done in [ea2f1ca](https://github.com/polylang/polylang/pull/1481/commits/ea2f1caf75970963f20d82f58fb72e7d6d5f3ebb).